setup.py

```diff
@@ -1,8 +1,9 @@
-from setuptools import setup
```

Comment:
This file can just be removed and options moved from setup.cfg to pyproject.toml.

The issues this was trying to fix is long solved upstream.